app/views/pugin/components/_header.html.haml

```diff
@@ -4,4 +4,4 @@
       .row
         .header__logo
           %a{:href => "/"}
-            %img{:alt => "parliament.uk", :src => "../images/parliament-uk--white-50.png", :srcset => "../images/parliament-uk--white-50.svg"}/
+            %img{:alt => "parliament.uk", :src => "http://ukpds.pugin-website.s3-website-eu-west-1.amazonaws.com/images/parliament-uk--white-50.png", :srcset => "http://ukpds.pugin-website.s3-website-eu-west-1.amazonaws.com/images/parliament-uk--white-50.svg"}/
```

Comment:
In fact - it should. When we go live are we really going to go through every URL and change it?